### PR TITLE
Game event update

### PR DESCRIPTION
**Pull request by TheSCREWEDSoftware**
### Description

- `game_event`
- `game_event_quest_condition`
- `game_event_condition`

Update headers, fixed re-directs, added Database to `game_event`

### Related Issue

Closes

## Thank you for contributing to the AzerothCore wiki.

Remember that the wiki is currently available in English and Spanish.

- https://www.azerothcore.org/wiki/home
- https://www.azerothcore.org/wiki/es/home

### CODE REVIEW

**Comment by TheSCREWEDSoftware** [comment]
Ready to be reviewed.

**Comment by sudlud** [comment]
Thanks for the PR! I'm not sure about copying the database table content into the wiki page - it's apparently going to get out of date and usually only the table structure is beeing described here.

File: docs/game_event.md [hunk 1]
```diff
@@ -1,66 +1,167 @@
...
 
 ### announce
+
+| Value | Description                                        |
+| ----- | -------------------------------------------------- |
+| 0     | Don't announce the Event                           |
+| 1     | Announce the description of the Event to the world |
+| 2     | Use the `event.announce` settings from the config  |
+
+### Database Content
```
Comment (Kitzunu) [inline docs/game_event.md:70]: delete this. The database isnt static and updating this everytime something changes is just bad.

**Review by Kitzunu — CHANGES_REQUESTED** [review]

File: docs/game_event.md (hunk 1 above)
Comment (TheSCREWEDSoftware) [reply docs/game_event.md]: Done.

**Pull request merged**